src/makeInfinityAdder.js

```diff
@@ -6,7 +6,23 @@
  */
 
 function makeInfinityAdder() {
-  // write code here
+  let amount = 0;
+
+  const adder = arg => {
+    if (arg === undefined) {
```

Comment:
Use `!arg`, its the same as `arg === undefined`

Reply:
I tried but that option didn't work when the argumen is zero